### ISSUE TITLE
win/installer: wintun driver uninstall doesn't work on windows/arm64

### ISSUE THREAD

**Issue by bradfitz** [tailscale/tailscale#1198]
From our NSIS installer's uninstall-old-version phase on a Windows ARM64 tablet....

```
Removing wintun adapter/driver...
panic: Error loading wintun.dll DLL: Unable to load library: The specified module could not be found.

goroutine 1 [running]:
github.com/tailscale/wireguard-go/tun/wintun.(*lazyProc).Addr(...)
         github.com/tailscale/wireguard-go@v0.0.20201119-0.20210116013233-4cd297ed5a7d/tun/wintun/dll_windows.go:56
github.com/tailscale/wireguard-go/tun/wintun.(*Pool).DeleteDriver(0x1920a400, 0x9, 0x1920a400, 0x0)
         github.com/tailscale/wireguard-go@v0.0.20201119-0.20210116013233-4cd297ed5a7d/tun/wintun/wintun_windows.go:179 +0xd8
tailscale.io/win/ui.doUninstall(0xec2020)
         tailscale.io/win/ui/main_windows.go:197 +0x1ea
tailscale.io/win/ui.Main()
         tailscale.io/win/ui/main.go:148 +0x6df
main.main()
         tailscale.io/win/wg32/main.go:13 +0x17
UninstallService: 2
Killing old tailscale-ipn.exe...
Kill: 603
Find: 603
Rename: tailscale-ipn-new.exe->tailscale-ipn.exe
```

/cc @alexbrainman @peske

**Comment by bradfitz** [comment]
@alexbrainman, ah, I see. Okay, that's less concerning then. And I can reproduce: I only get it on install, but the uninstaller works correctly.

So let's just `make tailscale-ipn.exe /uninstall` skip that step if the DLL doesn't exist on disk or (probably easier) if Windows.LzyProc.Find returns an error. (we'll have to copy/paste a few lines out of wireguard's definition of that LazyProc)

**Comment by alexbrainman** [comment]
win/ui: do not panic if wintun.dll is not found during installation

Fixes #1198

> let's just `make tailscale-ipn.exe /uninstall` skip that step if the DLL doesn't exist on disk or (probably easier) if Windows.LzyProc.Find

Done. See https://github.com/tailscale/winmin/pull/12

Alex
